src/DV7toDV8.sh

```diff
@@ -16,20 +16,57 @@
 
 # Keep working files generated during processing
 keepFiles=false
-
-while true
-do
-    case "$1" in
-    --keep-files)
-        echo "Option enabled to keep working files"
-        keepFiles=true
-        shift;;
-    "")
-        break;;
-    *)
-        targetDir=$1
-        shift;;
-    esac
+targetDir=$PWD # Default to current directory
+languageCodes=""
+languageCodeSet=false
+doviToolPath=""
+mkvextractPath=""
+mkvmergePath=""
+useLocal=false
+
+
+# Help function
+function print_help {
+    echo "Usage: $0 [options]"
+    echo ""
+    echo "Options:"
+    echo "-k|--keep-files       Keep working files"
+    echo "-t|--target PATH      Specify the target directory (default: current directory)"
+    echo "-l|languages LANG     Specify the language codes (comma-separated) for audio and subtitle tracks. If not specified, default to all tracks."
+    echo "-u|--use-local        Use local system binaries if available"
+    echo "-h|--help             Display this help message"
+    echo ""
+    exit 1
+}
+
+while (( "$#" )); do
+  case "$1" in
+  -k|--keep-files)
+    echo "Option enabled to keep working files"
+    keepFiles=true
+    shift;;
+  -h|--help)
+    print_help;;
+  -t|--target)
+    targetDir=$2
+    echo "Target directory set to '$targetDir'"
+    shift 2;;
+  -l|--languages)
+    languageCodes=$2
+    echo "Language codes set to '$languageCodes'"
+    languageCodeSet=true
+    shift 2;;
+  -u|--use-local)
+    useLocal=true
+    echo "Option enabled to use local binaries"
+    shift;;
+  -*|--*=) # unsupported flags
+    echo "Error: Unsupported flag $1" >&2
+    exit 1;;
+  *) # preserve positional arguments
+    PARAMS="$PARAMS $1"
```

Comment:
Unless you're moving the script or the mkv files into the same folder each time, the one required param for the script has been the target directory.

It's perfectly fine to support that use case, but what do you think about defaulting to the present working directory (as you've done) but then setting `targetDir` here, as the one unlabeled argument, rather than using the `-t`/`--target` label?

I think it's more common that you wouldn't be moving the files around, the script will have a home and your working files will be in different folders each time, so I think using an unlabeled arg for the target directory makes the most sense.

If `-t`/`--target` were to be required, then we'd need to update the Platypus app config file to reflect this change in the script as part of this PR.

Reply:
Updated. Let me know what you think 👍